backend/mora/common.py

```diff
@@ -135,6 +178,14 @@ def submit(self) -> str:
             return c.organisationfunktion.update(self.payload, self.uuid)
 
 
+def handler_for(obj: dict):
```

Comment:
Similarly here, this only concerns itself with org funks, despite the function being called `handler_for`. Rename it, or move the logic to the abstract org funk class somehow.